build-board-indexes.js

```diff
@@ -4,8 +4,9 @@ const fs = require('fs');
 glob('**/*.board.tsx', (error, boards) => {
     if (error) {
         // eslint-disable-next-line no-console
-        console.log(error);
+        console.error(error);
```

Comment:
shouldn't it throw?

Reply:
It's a design decision, I guess. Here, it'd run on every board file, and on the throw, the whole script would exit with an error. While with `console.error`, a message would go out, but processing of boards would still proceed, and this isn't stopping the npm script execution flow. 
Or something 🤷🏼